### ISSUE TITLE
Support for beta.2

### ISSUE THREAD

**Issue by jelleklaver** [c0deaddict/parcel-transformer-livescript#1]
Hi @c0deaddict, great work on bringing parcel and livescript together for v2, again!

It works like a charm for parcel 2.0.0-beta.1, but it seems that internal changes in `@parcel/source-map` broke it: 

```sh
ℹ️ Server running at http://localhost:1234
Building...
🚨 Build failed.
parcel-transformer-livescript: First parameter should be a string
TypeError: First parameter should be a string
    at NodeSourceMap.addRawMappings (./node_modules/parcel-transformer-livescript/node_modules/@parcel/source-map/dist/node.js:38:28)
    at Object.transform (./node_modules/parcel-transformer-livescript/lib/LiveScriptTransformer.js:38:13)
    at async Transformation.runTransformer (./node_modules/@parcel/core/lib/Transformation.js:830:5)
    at async Transformation.runPipeline (./node_modules/@parcel/core/lib/Transformation.js:585:36)
    at async Transformation.runPipelines (./node_modules/@parcel/core/lib/Transformation.js:413:40)
    at async Transformation.run (./node_modules/@parcel/core/lib/Transformation.js:316:19)
    at async Child.handleRequest (./node_modules/@parcel/workers/lib/child.js:255:9)
```

I dug into this a little deeper, but have no real understanding of parcel's transformers or the way source maps are created from livescript. The following code throws the exception (the line number don't exactly match, since `this.sourceMapInstance.addRawMappings` is collapsed into a single line in dist):

```js
/**
 * Local file: @parcel/source-map/dist/node.js:38:28
 * Github source: https://github.com/parcel-bundler/source-map/blob/master/src/node.js#L22-L29
 */

export default class NodeSourceMap extends SourceMap {
    ...
    addRawMappings(map: VLQMap, ...): SourceMap {
        let { sourcesContent, sources = [], mappings, names = [] } = map;
        ...
        this.sourceMapInstance.addRawMappings(
            mappings,
            sources.map((source) => (source ? relatifyPath(source, this.projectRoot) : '')),
            ...
        );
        return this;
    }
...
}
```

As far as I can tell, it's actually `relatifyPath` which expects a string input. In the previous version, it looked like this:
```js
/**
 * Github source: https://github.com/parcel-bundler/source-map/blob/31744d23c030e7bcdcf46041c1917c4824b3c183/src/SourceMap.js#L63
 */

export default class SourceMap {
    ...
    addRawMappings({mappings, sources, sourcesContent, names, lineOffset = 0, columnOffset = 0}): SourceMap {
        ...
        this.sourceMapInstance.addRawMappings(mappings, sources, sourcesContent, names, lineOffset, columnOffset);
        ...
    }
    ...
}
```
When I'm trying to run it from `test/cli`, I keep getting: `Could not find module "parcel-transformer-livescript", but it was listed in package.json. Run your package manager first.` So I haven't been able to get closer look at debugging yet.

I figured someone might quickly know what the issue is and find a simple resolution :-)

**Comment by c0deaddict** [comment]
Hello Jelle!

Thanks for testing this with the new beta.

After some debugging, I've found the issue: the source map output from Livescript should be converted `.toJSON()` before passing it on to Parcel. I fixed it in master with 3e9be33830905fcda8e47d4525705d7c43995eba.

Let me know if it also works for you.

**Comment by jelleklaver** [comment]
Thanks for the update, I just verified that your fix does work for us with `parcel@2.0.0-beta.2` as well! :-) 

I also know why I got `Could not find module "parcel-transformer-livescript"`, I had to compile the code first 🤦
